### PR TITLE
Added toggle entities/CL homepage

### PR DESCRIPTION
**Pull request by josepchetrit12**
# Description

This PR adds the toggle to show entities or Consensus clients.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,5 +19,8 @@
     "bugs": {
         "url": "https://github.com/migalabs/eth-metadashboard/issues"
     },
-    "homepage": "https://github.com/migalabs/eth-metadashboard#readme"
-}
\ No newline at end of file
+    "homepage": "https://github.com/migalabs/eth-metadashboard#readme",
+    "dependencies": {
+        "framer-motion": "^10.16.10"
```
Comment (josepchetrit12) [inline package.json:24]: @Artentii never add dependencies in the root directory, you should add it in the client driectory

File: packages/client/components/ui/ToggleHomepage.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { motion } from 'framer-motion';
+import { useState } from 'react';
+
+export default function ToggleHomepage() {
```
Comment (josepchetrit12) [inline packages/client/components/ui/ToggleHomepage.tsx:4]: @Artentii we prefer to use the arrow function, I have changed it to make all the code the same, example:

```js
const Test = () => {}
```

Comment (Artentii) [reply packages/client/components/ui/ToggleHomepage.tsx:4]: Okey, I'll start using the arrow function then.

File: package.json (hunk 1 above)
Comment (Artentii) [reply package.json:24]: Okey, I'll bare that in mind

**Pull request merged**